packages/server/index.js

```diff
@@ -1,17 +1,61 @@
-const { makeServer } = require('@conversationai/moderator-backend-core');
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+
+const url = process.argv.length >= 3 ? process.argv[2] : "http://localhost:8080";
+
+process.env.FRONTEND_URL = url;
+process.env.API_URL = `${url}/api`;
+
+const pUrl = new URL(url);
+const port = pUrl.port || (pUrl.protocol === 'https' ? 443 : 80);
+
+const expressWs = require('express-ws');
+const fs = require('fs');
+const https = require('https');
+const http = require('http');
+
+const { makeAppPart1, makeAppPart2 } = require('@conversationai/moderator-backend-core');
 const { mountWebFrontend } = require('@conversationai/moderator-frontend-web');
 const { mountAPI } = require('@conversationai/moderator-backend-api');
 
 async function init() {
-  const {
-    app,
-    start,
-  } = makeServer();
+  const app = makeAppPart1(false);
```

Comment:
`makeAppPart1` ins't a great name. If you have to use names like this, add a comment to explain what it does. Likewise with part2 below.